clippy_lints/src/methods/mod.rs

```diff
@@ -3887,6 +3888,27 @@ declare_clippy_lint! {
     "splitting a trimmed string at hard-coded newlines"
 }
 
+declare_clippy_lint! {
+    /// ### What it does
+    /// Checks for unnecessary calls to `min()`
+    ///
+    /// ### Why is this bad?
+    ///
+    /// In these cases it is not necessary to call `min()`
+    /// ### Example
+    /// ```no_run
+    /// let _ = 0.min(7_u32);
+    /// ```
+    /// Use instead:
+    /// ```no_run
+    /// let _ = 7;
```

Comment:
The minimum of 0 and 7 is 0, this description should be fixed.

Reply:
of course. *facepalm*. Thank you for catching it. I'll fix that